### PR TITLE
Qa 161

### PR DESCRIPTION
**Pull request by oleorhagen**

### CODE REVIEW

**Comment by oleorhagen** [comment]
> Content looks good, but strictly speaking, shouldn't these be done in the same commit? None of these commits will work independently.

Hmm, true. I made them as I was working, so each commit is a 'logical' change. I guess you are right though. But, as we're merging, the whole branch can still be reverted no?

**Comment by oleorhagen** [comment]
@kacf, I'm sorry, but you need to have another look.

On the plus side on your board, this means I did get the opportunity to squash the commits like you wanted.

On the negative side, there are three commits making small changes to the Makefile that you need to have a quick look at. The build was already broken prior to this PR. So to get the pipe green once again, I needed to fix this up :)

Sry for the trouble

-O

File: Makefile [hunk 1]
```diff
@@ -85,7 +85,7 @@ coverage:
 		rm -f coverage-tmp.txt;  \
 		$(GO) test -coverprofile=coverage-tmp.txt $$p ; \
 		if [ -f coverage-tmp.txt ]; then \
-			cat coverage-tmp.txt |grep -v 'mode:' >> coverage.txt; \
+			cat coverage-tmp.txt |  grep -v 'mode:' >> coverage.txt || /bin/true; \
```
Comment (kacf) [inline Makefile:88]: I don't get it, why has this started happening now? You didn't change any of the code, so the coverage should be the same as before. Which files are the problematic ones?

Comment (oleorhagen) [reply Makefile:88]: It has not started happening now. Checkout master and run the command. This was already broken. I just fixed it to get a green pipe. 

Do you want a separate PR for it?

Comment (kacf) [reply Makefile:88]: No, that's ok. Sorry, I missed the fact that it was already broken. Looks good then!

**Comment by oleorhagen** [comment]
@kacf I'm gonna have to ask you to have another look.

Migration with govendor was a pain, so I figured might as well do the go mod migration at the same time.

The updates also required some minor fixes to the `gitlab-ci.yml` file, courtesy of you (thx for the fix).

Does it still look okay?

**Pull request merged**